localisation/languages.json

```diff
@@ -4,7 +4,7 @@
     "description": "NZ English",
     "code": "en_nz",
     "flag": "🇳🇿",
-    "enabled": true
+    "enabled": false
```

Comment:
This shouldn't be changed in the default languages.json.

Any changes should come from snapshot.